lib/controllers/users.js

```diff
@@ -0,0 +1,44 @@
+const { Router } = require('express');
+const User = require('../models/User');
+const UserService = require('../services/UserService');
+const authenticate = require('../middleware/authenticate');
+const authorize = require('../middleware/authorize');
+
+const ONE_DAY_IN_MS = 1000 * 60 * 60 * 24; 
+
+module.exports = Router()
+  .post('/', async (req, res, next) => {
+    try {
+      const [, user] = await UserService.signUp(req.body);
```

Comment:
change this to `[token, user] = await UserService.signUp(req.body);` and then save the cookie the same way you do in the `/sessions` route
```js
         res
        .cookie(process.env.COOKIE_NAME, token, {
          httpOnly: true,
          maxAge: ONE_DAY_IN_MS,
        })
```